production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/lstchain_config_2024-09-17.json

```diff
@@ -304,13 +304,16 @@
         "spe_location": null,
         "pre_computed_multiplicity": 10
     },
+<<<<<<< HEAD
+=======
     "image_modifier": {
     "increase_nsb": true,
     "extra_noise_in_dim_pixels": 3.843,
     "extra_bias_in_dim_pixels": 1.07,
     "transition_charge": 8,
     "extra_noise_in_bright_pixels": 5.178
   },
+>>>>>>> 8c85c8a6cd11e8bbaf70eebfbb9ae4ab9215c1ed
```

Comment:
same here
the symbols `<<<<`, `====` and `>>>>` come from the git merge that you did not solve

Reply:
This is what makes the CI fail btw.

Reply:
Ok, I solved them by editing the json file myself, now it should be fine